src/atoms.ts

```diff
@@ -1,12 +1,14 @@
-import { Atoms } from "@guardian/content-api-models/v1/atoms";
-import { BlockElement } from "@guardian/content-api-models/v1/blockElement";
-import { Result, err, ok } from '@guardian/types/result';
-import { BodyElement, ElementKind } from "bodyElement";
-import { fromNullable } from '@guardian/types/option';
-import { DocParser } from "types/parserContext";
-import Int64 from 'node-int64';
-import { isValidDate } from "date";
+import type { Atoms } from "@guardian/content-api-models/v1/atoms";
+import type { BlockElement } from "@guardian/content-api-models/v1/blockElement";
+import { fromNullable } from "@guardian/types/option";
+import type { Result } from "@guardian/types/result";
+import { err, ok } from "@guardian/types/result";
+import type { BodyElement } from "bodyElement";
+import { ElementKind } from "bodyElement";
```

Comment:
This is interesting. So it's smart enough to figure out that `interface`s can be made into `type` imports, but `enum`s need to remain normal imports so that the lookups work? @sndrs

For context, `BodyElement` is an `interface` and `ElementKind` is an `enum`.

Reply:
I think this was one of the things `eslint --fix` couldn't automatically do so I had to do manually to fix the errors. It would convert to `import type` for all imports of a module, which wasn't working.

Reply:
yeah enums become actual JS don't they?

Reply:
Yeah, they output code, whereas something like an `interface` or a `type` only exists at compile time. It's interesting that the automated changes would have broken this 🤔

Reply:
would they? i thought it convert pure type imports to `import type` but left enum as a normal import? did i misdunderstand?

Reply:
> It would convert to `import type` for all imports of a module, which wasn't working.

Based on what @webb04 said above, yes? I believe this means that ESLint thought that _all_ type-related imports would become `import type`, including `enum`s, so he had to manually convert the ones that were `enum`s back?

Reply:
I can't remember if they were `enum`s or normal imports, but the eslint fix command did try to use `import type` for some modules that were not types. I think if one type was exported from a file, all the other exports were grouped as a type